Monika After Story/game/script-topics.rpy

```diff
@@ -10501,4 +10501,158 @@ label monika_hemispheres_nogets_snow:
     m 3hksdlb "At least you don't have to worry about shoveling it."
     call monika_snow_nogets_snow
     return
+    
+    default persistent._mas_pm_wants_children = None
+    # do you want children?
+    
+    default persistent._mas_pm_wants_1_child = None
+    # do you want one child?
+    
+    default persistent._mas_pm_wants_boy = None
+    # do you want a boy?
+    
+    default persistent._mas_pm_wants_girl = None
+    # do you want a girl?
+    
+    default persistent._mas_pm_wants_2_to_3_children = None
+    # do you want two to three children?
+    
+    default persistent._mas_pm_wants_many_children = None
+    # do you want many children?
+      
+init 5 python:
```

Comment:
Just a little note by the way. That's a ***lot*** of required sprites. Can you please go through your topic and see which ones you can change out for existing sprites? You can grab the dev expression previewer from the main repo, and throw it into your install. Codes in red are sprites that don't exist, those in pink do. Alternatively, if you're already a little familiar with how sprites look, you could just do a bit of ctrl+f work through `sprite-chart.rpy` and check if expressions exist there.